### DISCUSSION TITLE
Cursor disappears when moving left of scope line

### DISCUSSION THREAD

**Discussion by jwidauer** [lukas-reineke/indent-blankline.nvim#912] [Q&A] [answered]
Hi all!

Thanks a bunch for this great plugin! I've been using it for quite a while now and really love it!
There's one thing that I, unfortunately, can't seem to figure out. I don't think it's a bug, but I just can't figure out how to change it.
When my curser is within the section of a scope, it shows up normally, but when moving to the left of the scope separator line, it seems like the color is being set to the same as the background, making it invisible.

Here's a picture showing the cursor on the right side of the separator, showing normally:
![image](https://github.com/user-attachments/assets/5299d67a-89d0-4fd4-acc8-cccef54cb328)

And here's a picture from the same line, just with the cursor moved left past the separator column:
![image](https://github.com/user-attachments/assets/2f457d44-443a-49f9-9986-2a137f87b729)

Would anybody be able to tell me what highlight should be applied there? Or how I could change this?
I'm using the "darker" version of the [onedark](https://github.com/navarasu/onedark.nvim) theme, if that helps at all.
I think I mostly just don't know which one of the thousands of settings to look for under `:highlight` or in the theme description.

Any and all help would be greatly appreciated! Thanks in advance!

**Comment by j-krl** [comment]
I think this is a bug. When I have my cursor on top of one of the indent lines in insert mode, it disappears. I don't think the colour of the cursor is relevant as you can see below. First pic is normal mode, second is insert mode.

<img width="194" alt="Screenshot 2024-08-03 at 3 08 28 AM" src="https://github.com/user-attachments/assets/34ca7165-d382-40c9-9d29-d0636b36db2f">
<img width="220" alt="Screenshot 2024-08-03 at 3 08 38 AM" src="https://github.com/user-attachments/assets/c39e3b97-2b0e-49b0-a004-d4f7310e108e">

**Comment by j-krl** [comment]
Update: Actually looking like this is a Terminal issue. It's happening on iTerm2 but not on Kitty. I'll have to investigate.

**Comment by lukas-reineke** [comment, marked as answer]
This is not a bug in IBL.
IBL has no control over the cursor at all. This is Neovim core, or your terminal emulator.

**Reply by jwidauer** [reply]
That's why I said that I don't think it's a bug in IBL, but I can't figure out how to change it.
And since I've seen a couple of discussions / issues related to this in this repo, I was hoping that somebody (or you?) would be able to know how I could change it or what Neovim setting might affect this.
